mod/wet4/views/default/page/elements/login_menu.php

```diff
@@ -195,18 +195,24 @@
 
 <script>
 $(document).ready(function () {
-$('#signin-link').click(function () {
-    $('#signin-dropdown').toggle();
-    $('#session').toggleClass('active');
-    return false;
-});
-$('#signin-dropdown').click(function(e) {
-    e.stopPropagation();
-});
-$(document).click(function() {
-    $('#signin-dropdown').hide();
-    $('#session').removeClass('active');
-});
+    $('#signin-link').click(function () {
+        $('#signin-dropdown').toggle();
+        $('#session').toggleClass('active');
+        return false;
+    });
+
+    $('#signin-dropdown').click(function(e) {
+        e.stopPropagation();
+    });
+
+    $(document).click(function() {
+        $('#signin-dropdown').hide();
+        $('#session').removeClass('active');
+    });
+
+    $('#login_focus').click(function(){
```

Comment:
When I tab to the login and hit enter it does not perform this focus (It does when I click with the mouse though). Perhaps try adding this focus function to the link $('#signin-link')